packages/swing-store/src/snapStore.js

```diff
@@ -405,8 +403,12 @@ export function makeSnapStore(
    */
   function deleteVatSnapshots(vatID) {
     ensureTxn();
-    for (const rec of sqlGetSnapshotList.iterate(vatID)) {
-      const exportRec = snapshotRec(vatID, rec.endPos, undefined);
+    const deletions = [];
```

Comment:
this does the job, but as a style nit I'd go with `Array.from(sqliteGetSnapshotList.iterate(vatID))`

Reply:
Good catch.  Even better, though, is:
```const deletions = sqlGetSnapshotList.all(vatID);```
which I have done.